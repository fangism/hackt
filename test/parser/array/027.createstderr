ERROR: Index [7] of int_array_1D blah already instantiated!
Error encountered during module::create.
ERROR in creating.  Aborting.
