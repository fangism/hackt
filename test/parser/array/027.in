// "027.in" should fail: overlapping sparse arrays

int blah[5..10];
int blah[7..7];

