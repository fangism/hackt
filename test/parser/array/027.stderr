Detected overlap in the sparse collection for blah, precisely: [7..7].  ERROR!  [4:5..8]
